bedrock/firefox/templates/firefox/welcome/page15/base.html

```diff
@@ -0,0 +1,202 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{% endblock %}
+
+{% block page_image %}{{ static('img/products/vpn/common/social-share.png') }}{% endblock %}
+
+{% block body_class %}welcome-page15{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_welcome_page15') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-15' %}
+{% set _utm_campaign = 'welcome-15-vpn' %}
+
+{% block experiments %}
+  {% if switch('welcome-15-experiment', ['en-US', 'de', 'fr']) %}
+    {{ js_bundle('firefox_welcome_page15_experiment') }}
+  {% endif %}
+{% endblock %}
+
+{% block shoulder_logo %}
+  {{ picture(
+    url="protocol/img/logos/firefox/browser/logo-word-hor.svg",
+    sources=[
+      {
+        "media": "(prefers-color-scheme: light)",
+        "srcset": {
+          "protocol/img/logos/firefox/browser/logo-word-hor.svg": "default",
+        }
+      },
+      {
+        "media": "(prefers-color-scheme: dark)",
+        "srcset": {
+          "protocol/img/logos/firefox/browser/logo-word-hor-white.svg": "default",
+        }
+      }
+    ],
+    optional_attributes={
+      "class": "c-header-logo",
+      "alt": "Firefox",
+      "width": "216",
+      "height": "40",
+    }
+  ) }}
+{% endblock %}
+
+{% block shoulder_cta %}{% endblock %}
+
+{% block content_intro %}
+    {% if variation == '2' %}
+      <div class="c-main variation-2 mzp-l-content mzp-t-content-lg">
+        <img class="c-main-logo" src="/media/protocol/img/logos/mozilla/vpn/logo-flat.svg" width="60" alt="">
+        <h1 class="c-main-title">{{ var_1_main_title }}</h1>
```

Comment:
Sorry, just noticed all the strings are labeled "var_1" but this is actually `variation == 2` above. Should they be switched around? Just to make sure we have the right text on the right variant.

Reply:
Doh, will fix!